lib/binary_search_tree.js

```diff
@@ -0,0 +1,149 @@
+var BinaryNode = require('../lib/binary_node.js');
+
+var BinaryTree = function (value){
+  this.root = new BinaryNode(value);
+  this.a = ""; //string
+  this.s = 0;  //size
+  this.d = 0;  //depth
+  this.b = 0;  //balance
```

Comment:
These are not very meaningful var names. If you use the words, you won't need the comments. :]